release.sh

```diff
@@ -658,13 +658,18 @@ function publish_artifacts() {
 
 # Sets the github release with the highest semver to 'latest'
 function set_latest_to_highest_semver() {
-  local last_version # don't combine with the line below, or $? will be 0
+  if ! (( PUBLISH_TO_GITHUB )); then
```

Comment:
if we move this call to the publish to github function then that function already has this logic there